### ISSUE TITLE
Inconsistent minifying in browser polyfills

### ISSUE THREAD

**Issue by romainmenke** [csstools/postcss-plugins#220]
Some inconsistencies I noticed regarding these:
* `css-has-pseudo` and `css-prefers-color-scheme` ship with all their files minified; `css-blank-pseudo` does not.
* `css-prefers-color-scheme` ships with `browser.min.js` but this file is identical to `browser.js` (as both are minified).

I don't really have an opinion about whether they should be minified or not (being able to read the code is nice if something goes wrong but these polyfills should be pretty much invisible anyway) but they should probably be consistent :)

_Originally posted by @ehoogeveen-medweb in https://github.com/csstools/postcss-plugins/issues/186#issuecomment-1026680916_

**Comment by romainmenke** [comment]
@ehoogeveen-medweb I moved this into it's own issue as it's not related to changing a package name :)

Can you provide urls for these examples?
They should all be minified.

**Comment by ehoogeveen-medweb** [comment]
Hmm, I'm not sure what's the best way to link files from the published packages, but here are some RunKit links:

`css-blank-pseudo` (not minified):
* https://npm.runkit.com/css-blank-pseudo/browser.js
* https://npm.runkit.com/css-blank-pseudo/dist/browser.cjs
* etc.

`css-prefers-color-scheme` (redundant browser.min.js):
* https://npm.runkit.com/css-prefers-color-scheme/browser.js
* https://npm.runkit.com/css-prefers-color-scheme/browser.min.js

**Comment by romainmenke** [comment]
Found it : https://github.com/csstools/postcss-plugins/blame/main/rollup/presets/browser.javascript.js

We only "recently" added `terser` to the browser polyfills and some of the packages haven't needed a release yet.

So over time all will be minified.
These also have sourcemaps which is why minifying shouldn't prevent users from debugging issues.

The reason that there are inconsistencies in naming and location of files is because each package had it's own way of doing this.

We recently migrated to a mono repo and everything now uses a single build flow.
To make this non-breaking we simply copy files from the new build flow to old locations.

If a package previously had minified and unminified versions they will actually be the same file but copied over to make it non-breaking.

Over time we will clean this up.
But we want to avoid disruption for users where possible.
